### DISCUSSION TITLE
question about a command

### DISCUSSION THREAD

**Discussion by cousc0us** [AntiCope/meteor-rejects#128] [bruh] [answered]
What's ore sim and whats the difference of ore sim and normal xray?

**Comment by C10udburst** [comment, marked as answer]
ore sim uses the world seed to find where the ores should be, completely ignoring what server sends. on the other hand xray just displays what server sends. the result of that is that oresim bypasses "antixray" plugins, but it won't work when the server uses a different world generation or you don't know the seed.
